### PR TITLE
Edit chaos collector database deployment to allow Affinity and Tolerations

### PR DESCRIPTION
**Pull request by tfulcrand**
### What problem does this PR solve?

The chaos collector database don't use `affinity` and `tolerations` dashboard variables defined in values.yaml

### What is changed and how does it work?

Being able to use the `affinity` and `tolerations` for Chaos collector database based on dashboard values in values.yaml

### Does this PR introduce a user-facing change?:

```release-note
Allow users to configure affinity and tolerations for Chaos Collector database
```

### CODE REVIEW

**Comment by cwen0** [comment]
@tfulcrand Thanks for your contribution, please sign CLA first.

**Comment by zhouqiang-cl** [comment]
@tfulcrand Thank you for your contribution, could you please sign CLA

File: helm/chaos-mesh/templates/collector-mysql.yaml [hunk 1]
```diff
@@ -50,6 +50,14 @@ spec:
         - name: data
           persistentVolumeClaim:
             claimName: collector-mysql-pv-claim
+      {{- with .Values.dashboard.affinity }}
```
Comment (cwen0) [inline helm/chaos-mesh/templates/collector-mysql.yaml]: Can you add `nodeSelector` field for this Deployment at here?

Comment (tfulcrand) [reply helm/chaos-mesh/templates/collector-mysql.yaml]: Thanks for your comment : `nodeSelector` is added now.

**Comment by cwen0** [comment]
@yeya24 PTAL

**Pull request merged**